index.html

```diff
@@ -77,6 +77,39 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
 
+
+<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
It looks like you've added your code, directly below the template. 
Please refer to the instructions and move it, so it would be the most recently added card. (should be line 118).